API

```diff
@@ -27,24 +27,24 @@ remove_rownames(.data)
 repair_names(x, prefix = "V", sep)
 rowid_to_column(.data, var = "rowid")
 rownames_to_column(.data, var = "rowname")
-set_tidy_names(x, syntactic = FALSE, quiet = FALSE)
+set_tidy_names(x, quiet = FALSE)
 tbl_sum(x)
-tibble(..., .rows = NULL, .tidy_names = NULL)
+tibble(..., .rows = NULL, .name_repair = NULL)
 tibble_(xs)
-tidy_names(name, syntactic = FALSE, quiet = FALSE)
+tidy_names(name, quiet = FALSE)
 tribble(...)
 trunc_mat(x, n = NULL, width = NULL, n_extra = NULL)
 view(x, title = NULL, ...)
 
 ## Own S3 methods
 
 as_tibble.NULL(x, ...)
-as_tibble.data.frame(x, validate = TRUE, ..., .rows = NULL, .tidy_names = if (validate) NULL else FALSE, rownames = pkgconfig::get_config("tibble::rownames", NULL))
+as_tibble.data.frame(x, validate = TRUE, ..., .rows = NULL, .name_repair = NULL, rownames = pkgconfig::get_config("tibble::rownames", NULL))
 as_tibble.default(x, ...)
-as_tibble.list(x, validate = TRUE, ..., .rows = NULL, .tidy_names = if (validate) NULL else FALSE)
+as_tibble.list(x, validate = TRUE, ..., .rows = NULL, .name_repair = NULL)
 as_tibble.matrix(x, ...)
 as_tibble.poly(x, ...)
-as_tibble.table(x, `_n` = "n", ..., n = `_n`)
+as_tibble.table(x, _n = "n", ..., n = _n)
```

Comment:
This needs https://github.com/r-lib/pkgapi/pull/26, but I can fix after merging.